webapp/src/plugin_constants/common.ts

```diff
@@ -6,9 +6,12 @@ export const RightSidebarHeader = 'Azure DevOps';
 export const MMCSRF = 'MMCSRF';
 export const HeaderCSRFToken = 'X-CSRF-Token';
 
-export enum ToggleSwitchLabelPositioning {
-    Left = 'left',
-    Right = 'right',
-}
+export const boardsEventTypeMap: Record<EventType, string> = {
+    create: 'Work Item Created',
+    update: 'Work Item Updated',
+    delete: 'Work Item Deleted',
+};
 
-export const ToggleLabel = 'Show all subscriptions';
+export const channelType = {
```

Comment:
This is not needed. There's already a constant for this in the `mattermost-redux` package.